### PR TITLE
#681@minor: Add scrollX, scrollY, pageXOffset, pageYOffset to window.

### PR DESCRIPTION
**Pull request by Ben2HellAndBack**
Add [scrollX](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollX), [scrollY](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY), [pageXOffset](https://developer.mozilla.org/en-US/docs/Web/API/Window/pageXOffset) and [pageYOffset](https://developer.mozilla.org/en-US/docs/Web/API/Window/pageYOffset)  to `Window`

### CODE REVIEW

**Comment by Mas0nShi** [comment]
Fixes #680

File: packages/happy-dom/src/window/Window.ts [hunk 1]
```diff
@@ -483,6 +483,42 @@ export default class Window extends EventTarget implements IWindow {
 		this.document._onWindowReady();
 	}
 
+	/**
+	 * The number of pixels that the document is currently scrolled horizontally
+	 *
+	 * @returns number
+	 */
+	public get scrollX(): number {
+		return this?.document?.documentElement?.scrollLeft ?? 0;
```
Comment (capricorn86) [inline packages/happy-dom/src/window/Window.ts:492]:
```suggestion
		return this.document.documentElement?.scrollLeft ?? 0;
```

File: packages/happy-dom/src/window/Window.ts [hunk 2]
```diff
@@ -483,6 +483,42 @@ export default class Window extends EventTarget implements IWindow {
...
+		return this.scrollX;
+	}
+
+	/**
+	 * The number of pixels that the document is currently scrolled vertically
+	 *
+	 * @returns number
+	 */
+	public get scrollY(): number {
+		return this?.document?.documentElement?.scrollTop ?? 0;
```
Comment (capricorn86) [inline packages/happy-dom/src/window/Window.ts:510]:
```suggestion
		return this.document.documentElement?.scrollTop ?? 0;
```

**Review by capricorn86 — CHANGES_REQUESTED** [review]
Thanks for your contribution @Ben2HellAndBack! :star2:  Just found some very minor things.

**Comment by Ben2HellAndBack** [comment]
NP! @capricorn86 all fixed up now 👍

**Review by capricorn86 — APPROVED** [review]

**Pull request merged**